src/sampler.jl

```diff
@@ -37,40 +37,43 @@ sample(
     τ::AbstractProposal,
     θ::AbstractVector{T},
     n_samples::Int,
-    adaptor::Adaptation.AbstractAdaptor,
+    adaptor::Union{Nothing,Adaptation.AbstractAdaptor}=nothing,
```

Comment:
Can we define a special `NoAdaptation <: AbstractAdaptor ` instead of using `nothing`?

Reply:
Added in https://github.com/TuringLang/AdvancedHMC.jl/pull/65/commits/a0e1a9f770b38b7bee57a10aab17be950c065ea6